.github/workflows/ontology_diff.yml

```diff
@@ -0,0 +1,70 @@
+name: Post Markdown Comment on Pull Request
+
+on:
+    workflow_dispatch:
+    pull_request:
+        types: [opened, synchronize]
+
+jobs:
+    post-comment:
+        runs-on: ubuntu-latest
+
+        steps:
+        - name: Checkout repository
+          uses: actions/checkout@v4
+
+        - name: Set up Python 3.10
+          uses: actions/setup-python@v5
+          with:
+            python-version: '3.10'
+
+        - name: Install oaklib
+          run: pip install git+https://github.com/INCATools/ontology-access-kit.git@release-notes-emit
+
+        - name: Run make diff-md
+          run: |
+            cd src/ontology
+            make diff-md
```

Comment:
This diff is not the correct one atm, as you don't want to diff with the release version of the ontology, but the `main branch` version. 

Maybe rather than working on this, you should consider to meet with @anitacaron to discuss how to fold this in with 

https://github.com/INCATools/ontology-development-kit/blob/master/template/_dynamic_files.jinja2#L818

This is where this needs to go anyways! Maybe ask her on slack what her recommendation is.

Reply:
Done. I don't understand what to do with jinja. @anitacaron , any thoughts?

Reply:
1. copy paste the entire action
2. Swap out two rows:
https://github.com/INCATools/ontology-development-kit/blob/e10366347bff8a572ec6a7f3daf3382da603ee8c/template/_dynamic_files.jinja2#L895C1-L896C1
https://github.com/INCATools/ontology-development-kit/blob/e10366347bff8a572ec6a7f3daf3382da603ee8c/template/_dynamic_files.jinja2#L842

This one probabbly needs your truncation logic? Maybe there already is some truncation logic implemented, not sure:
https://github.com/INCATools/ontology-development-kit/blob/e10366347bff8a572ec6a7f3daf3382da603ee8c/template/_dynamic_files.jinja2#L912

Reply:
For the release diff, we only add a link to the md file. Here's the example in CL https://github.com/obophenotype/cell-ontology/blob/2180772b156cf79797b04cc494efc8185088428c/.github/workflows/post-release-diff.yaml#L22